packages/test/test-version-utils/src/compatUtils.ts

```diff
@@ -102,7 +103,7 @@ function filterRuntimeOptionsForVersion(
 	if (version.startsWith("1.")) {
 		options = {
 			// None of these features are supported by 1.3
-			compressionOptions: undefined,
+			compressionOptions: disabledCompressionConfig,
```

Comment:
@vladsud -- FYI, to disable compression pass this not undefined

Reply:
why do we need to do this? why can't we just handle this internally?

Reply:
as this will be a breaking change for partners, and AFIK this was not called out anywhere before now

Reply:
We were queued up to remove the option altogether.  Pivoting to make the behavior change the break.

Reply:
> why can't we just handle this internally?

You mean just not compress even if it's requested via options?

Reply:
i just don't understand this change and why it is necessary?

Reply:
> as this will be a breaking change for partners

This is for compat testing, not sure who this will break.  The function is `@internal` too

Reply:
so i only replied here, but  i see a bunch of changes that do the same thing? did the tests just not work? i'm just confused why we need any other changes besides just adding an error for the case we are explicitly disallowing, and i worry we are doing something that necessitates the other changes, and i don't know for certain it is safe to do

Reply:
to me, i think enableGroupBatching == false + undefined compressionOptions should just work, and compression should just be disabled. only explicit enableGroupBatching === false and defined compressionOptions  should fail

Reply:
Thanks for the suggestion, I agree that will be better.  Updating as applicable throughout, but leaving this one explicitly disabling.